docs/source/contributing/adding_workloads.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
Curious, why are we adding this docs in Jupyter notebooks format?

Reply:
Main idea was that the notebook could be downloaded and used by users interactively as an example. Plus it renders pretty nice within sphinx and makes adding in large code blocks that should be executable fairly easy.

Reply:
Additionally I wanted to test it out- it was pretty fun!

Reply:
Here is what the rendered page looks like: https://my-benchmark-wrapper-fork.readthedocs.io/en/latest/contributing/adding_workloads.html. Adding in all of the code, and the code's output was super easy to do and test within a notebook.